apps/admin/lib/useMe.js

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@apollo/client'
+import { ME_QUERY } from './withMe'
+import { useEffect } from 'react'
+import { setUser as setSentryUser } from '@sentry/nextjs'
+
+export function useMe() {
+  const { data } = useQuery(ME_QUERY)
+
+  // set sentry user
+  useEffect(() => {
+    if (data?.me) {
+      setSentryUser({ id: data?.me.id })
+    } else {
+      setSentryUser(null)
+    }
+
+    return () => {
+      setSentryUser(null)
+    }
+  }, [data?.me])
```

Comment:
I don't like this as a side effect of useMe. This way, it also doesn't seem to work when the `withMe` HOC is used. So, preferably, the Sentry user should be set explicitly in _app.tsx.

Same concern applies to publikator app.

Reply:
Also, I don't think useEffect should be used? It won't work on the server …